tests/unit/test_trim_path_prefix.py

```diff
@@ -0,0 +1,38 @@
+# -----------------------------------------------------------------------------
+#                     The CodeChecker Infrastructure
+#   This file is distributed under the University of Illinois Open Source
+#   License. See LICENSE.TXT for details.
+# -----------------------------------------------------------------------------
+
+""" Test cases to trim a path by using prefixes. """
+
+import unittest
+
+from libcodechecker.util import trim_path_prefixes
+
+
+class TrimPathPrefixTestCase(unittest.TestCase):
+    """
+    Test cases to trim a path by using prefixes.
+    """
+
+    def test_no_prefix(self):
```

Comment:
I miss test cass like:
```
/a/b/c/foo.txt
/a/b/car/foor.txt
/a/b/common.txt
```
After removing /a/b/c prefix we should preserve the c in car folder and common.txt